<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculadora</class>
 <widget class="QMainWindow" name="Calculadora">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>699</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>85</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>85</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>85</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calculadora</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>download.jpg</normaloff>download.jpg</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="DisplayLCD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="digitCount">
     <number>10</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Digito7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Sete &quot;Atalho: 7&quot;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="shortcut">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Digito8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Oito &quot;Atalho: 8&quot;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="shortcut">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Digito9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Nove &quot;Atalho: 9&quot;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="shortcut">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Digito5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Cinco &quot;Atalho: 5&quot;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="shortcut">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Digito6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Seis &quot;Atalho: 6&quot;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="shortcut">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Digito4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Quatro &quot;Atalho: 4&quot;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Digito1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="statusTip">
     <string>Um &quot;Atalho: 1&quot;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Digito2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Dois &quot;Atalho: 2&quot;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Digito3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Três &quot;Atalho: 3&quot;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Virgula">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Vírgula &quot;Atalho: ,&quot;</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
    <property name="shortcut">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calcular">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>580</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Igual &quot;Atalho: Enter&quot;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="shortcut">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Digito0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>580</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Zero &quot;Atalho: 0&quot;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="shortcut">
     <string>0</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Multiplicar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Multiplicar &quot;Atalho: *&quot;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="shortcut">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Subtrair">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>580</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Subtrair &quot;Atalho: -&quot;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="shortcut">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Somar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Somar &quot;Atalho: +&quot;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="shortcut">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dividir">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Dividir &quot;Atalho: /&quot;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="shortcut">
     <string>/</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Memoria">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Fracao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Inverte número &quot;Atalho: D&quot;</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
    <property name="shortcut">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Apagar_Tudo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Apagar tudo &quot;Atalho: A&quot;</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string>CA</string>
    </property>
    <property name="shortcut">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Apagar_Numero">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Aapaga número &quot;Atalho: Del&quot;</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="shortcut">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Apaga_Digito">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Apaga dígito: &quot;Atalho: BackSpace&quot;</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Inverter_sinal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Inverte sinal &quot;Atalho: I&quot;</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
    <property name="shortcut">
     <string>I</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Elevado">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Elevado &quot;Atalho: E&quot;</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
    <property name="shortcut">
     <string>E</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Pi">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Número pi &quot;Atalho: P&quot;</string>
    </property>
    <property name="text">
     <string>π</string>
    </property>
    <property name="shortcut">
     <string>P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Logaritmo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Logarítimo &quot;Atalho: L&quot;</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
    <property name="shortcut">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Seno">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Seno &quot;Atalho: S&quot;</string>
    </property>
    <property name="text">
     <string>sen</string>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Tangente">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Tangente &quot;Atalho: T&quot;</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
    <property name="shortcut">
     <string>T</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Fatorial">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Fatorial &quot;Atalho: F&quot;</string>
    </property>
    <property name="text">
     <string>fat</string>
    </property>
    <property name="shortcut">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cosseno">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Cosseno &quot;Atalho: C&quot;</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
    <property name="shortcut">
     <string>C</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Modo_Calculadora">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigura_es">
    <property name="title">
     <string>Configurações</string>
    </property>
    <addaction name="separator"/>
    <addaction name="Apagar_Historico"/>
    <addaction name="Configurar"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
    <addaction name="Versao"/>
   </widget>
   <addaction name="menuConfigura_es"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Apagar_Historico">
   <property name="text">
    <string>Apagar Histórico</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="Versao">
   <property name="text">
    <string>Versão</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="Configurar">
   <property name="text">
    <string>Configurar</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
